doc/user-guide/src/SUMMARY.md

```diff
@@ -3,9 +3,9 @@
 [Introduction](index.md)
 - [Installation](installation/index.md)
     - [Windows](installation/windows.md)
-        - [MSVC Prerequistes](installation/windows-msvc.md)
+        - [MSVC prerequistes](installation/windows-msvc.md)
     - [Other installation methods](installation/other.md)
-    - [Package managers](installation/package-managers.md)
+    - [Adopting external toolchains](installation/adopting-external-toolchains.md)
```

Comment:
I found `package-managers` ambiguous since it could mean
installing `rustup` using a package manager,
or installing `rustup` while already using externally-managed
`rust` toolchains.

Hopefully this change will make that ambiguity disappear.

Reply:
Can you please add a redirect when renaming a file? 

In `book.toml`, you can add something like:

```toml
[output.html.redirect]
"/installation/package-managers.html" = "adopting-external-toolchains.html"
```

(I think that syntax is correct, but please test it.)

Reply:
Will do.

Reply:
@ehuss Thanks a lot! I tested it locally and it works without any problem.

Reply:
Hmm, not completely sure this feels like an improvement. "Package manager" is a clear existing term of art, whereas "external toolchain" seems like a term you invented. Maybe there's a better way to disambiguate?

Reply:
@djc I originally used something like "Adopting toolchain installations unmanaged by `rustup`" but it was a bit too long. Any suggestions?

Reply:
That seems pretty good. Maybe just drop the "installations" part?

Reply:
I slightly prefer "unmanaged" over "external" since it seems little more self-explanatory, but not sure how we decide on brevity/layout vs being explicit.

Reply:
cc @rbtcollins

Reply:
I think that that doc really deals with two things.

One is about toolchains, but the other, bigger one, is that rustup refuses to install if someone already has rust installed via any other method.

I can understand that 'package managers' is too abbreviated, but it is honestly 90+% of the bug reports we get from that space - Windows and Mac users seem to find rustup themselves; folk on Linux tend to apt-get or whatever and then when the version isn't satisfactory, or something else specifically wants rustup, then try to install rustup.

Perhaps a title of 'Installing when rust is already installed' ?

Reply:
@rbtcollins How about making it a bit shorter: "Already installed Rust?`

Reply:
If you still think this is not appropriate, please feel free to un-resolve this conversation :)